source/_integrations/tami4.markdown

```diff
@@ -0,0 +1,18 @@
+---
+title: "Tami4 Edge / Edge+"
+description: "Tami4 Edge Integration"
+ha_release: "2022.11"
+ha_category: Other
```

Comment:
```suggestion
ha_category: Sensor
```

Reply:
It's not exactly a sensor, it also controls the device

Reply:
This attribute can be a list, on your current PR there is only sensor platform, you can still add other platforms later